builder/builder.go

```diff
@@ -182,12 +182,12 @@ func (b *Builder) runStep(ctx context.Context, step *graph.Step) error {
 				workDir = step.ID
 			}
 
-			step.Cmd = replacePositionalContext(step.Cmd, ".")
+			step.Build = replacePositionalContext(step.Build, ".")
 		}
 
 		args = b.getDockerRunArgs(volName, workDir, step)
-		args = append(args, "docker")
-		args = append(args, strings.Fields(step.Cmd)...)
+		args = append(args, "docker", "build")
+		args = append(args, strings.Fields(step.Build)...)
```

Comment:
>(args, strings.Fields(step.Build) [](start = 15, length = 33)

Are users already be able to to pass arbitrary arguments to build and run commands?

Reply:
yes